translations/town.tex/pl.po

```diff
@@ -80,7 +80,21 @@ msgid ""
 "Some Building Tiles are double-sided, and may later be upgraded and flipped to represent two different buildings at the same time. Such upgrades must be \\textbf{Built in order}.\\par\n"
 "\n"
 msgstr ""
-
+"\\hypertarget{Miasto}{Każda}\\index{Miasto} frakcja ma własne miasto, które znajduje się na środku swojego kafla początkowego.\n"
```

Comment:
W oryginale starting tile to kafelek początkowy + imo frakcja, miasto wielką literą

```suggestion
"\\hypertarget{Miasto}{Każda}\\index{Miasto} Frakcja ma własne Miasto, które znajduje się na środku swojego Kafelka Początkowego.\n"
```